enterprise-suite/tests/smoke_prometheus

```diff
@@ -230,6 +232,19 @@ busy_wait app_data
 
 prom_has_data "$app_instances"
 
+# automatic metrics should gain an es_monitor_type label when a custom monitor is created
+curl -XPOST -H "Content-Type: application/json" -H 'Author-Name: Me' -H 'Author-Email: me@lightbend.com' -H 'Message: testing!' \
+    "$ES_MONITOR_API/monitors/es-test/mytest" \
+    --data '{"model":"threshold","parameters":{"name":"my_custom_monitor","metric":"up{es_workload=\"es-test\"}","comparator":"<","threshold":"1","window":"5m","confidence":"0.5","severity":"critical","summary":"pod not up","description":"","monitorType":"es-test"}}'
```

Comment:
shouldn't need `es_workload="es-test"` as a filter.  Scope will handle that.

Reply:
in refactored API, scope will come from `"$ES_MONITOR_API/monitors/es-test/mytest"` and set scope to "es-test" for you, so you also won't need to set monitorType = scope like this. But we need the refactor to land, and then tweak this a little.